tools/image-info

```diff
@@ -2519,11 +2524,15 @@ def discover_lvm(dev:str, parent:devices.Device, devmgr:devices.DeviceManager):
         }
 
         # Create an OSBuild device object for the LVM partition
+        info = index.get_module_info("Device", "org.osbuild.lvm2.lv")
         device = devices.Device(
-                    name,
-                    index.get_module_info("Device", "org.osbuild.lvm2.lv"),
-                    parent,
-                    options)
+            name,
+            info,
+            parent,
+            options)
+        if not info:
+            raise RuntimeError("can't find org.osbuild.lvm2.lv")
```

Comment:
This check should be above the device creation, but I suppose it's not a big issue since the Device constructor just assigns it to the property.